<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="opacity_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="value_opacity">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_fill_color">
       <property name="text">
        <string>Change Fill Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stroke_color">
       <property name="text">
        <string>Change Stroke Color</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Stroke Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="stroke_width_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="value_stroke_width">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="index_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>scale_x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="scale_x">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="value_scale_x">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>scale_y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="scale_y">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="value_scale_y">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="scale_lock">
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionEllipse"/>
    <addaction name="actionCursor"/>
    <addaction name="actionMove"/>
   </widget>
   <addaction name="edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="text">
    <string>Cursor</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
